### PR TITLE
update binary.md :memo:

### PR DESCRIPTION
**Pull request by abhinavsri360**
Insert description of PR here
I have reframed the content for Binary Numbers and made that interactive.
Changed the word 'add' and replaced it with 'carry' as both have different meanings and cannot be used interchangeably.
Added a note nearby simulator that the digit changes from 0 -> 1 only after clicking them as this might become a bug later as people cannot judge by watching to what they see.
Potential applicant for GSoD '20 😄
---

## ✅️ By submitting this PR, I have verified the following

* [x] Checked to see if a similar PR has already been opened 🤔️
* [x] Reviewed the contributing guidelines 🔍️

### CODE REVIEW

**Comment by abhinavsri360** [comment]
@YashKumarVerma here is my content changes.

File: docs/binary.md [hunk 1]
```diff
@@ -58,7 +67,7 @@ It is just like counting in decimal except we reach 10 much sooner.
 | Decimal       | Explanation            |
 |:-------------:|:----------------------:|
 |	99	 |	When we run out of digits, we ...|
-|	100	 |	... start back at 0 again, but add 1 on the left|
+|	100	 |	... start back at 0 again, but carry 1 on the left|
```
Comment (Shivansh2407) [inline docs/binary.md:70]: Make this `but carry 1 on the left` bold as done in other tables

File: docs/binary.md [hunk 2]
```diff
@@ -13,10 +13,19 @@ nav_order: 2
...
+represent are made up of only two digits i.e., 0 and 1 which is quite interesting.
+Let us show you the binary representation of a decimal number(The numbers which we use for counting
+i.e., from 0-9) in binary.
+Example:
+Decimal number :: 25
+Binary Number :: 11001
+
+Note ::
 There is no 2, 3, 4, 5, 6, 7, 8 or 9 in Binary!
+Also, each digit in binary representation is either 0 or 1.
```
Comment (Shivansh2407) [inline docs/binary.md:28]: You have used this fact in the above paragraph. Please change this.

File: docs/binary.md [hunk 3]
```diff
@@ -39,15 +48,15 @@ It is just like counting in decimal except we reach 10 much sooner.
 | 1            | Then 1                 |
 | 2-8          | Count 1,2,3,4,5,6,7,8  |
 | 9            | This is the **last digit** in Decimal|
-|**1**0        | So we start back at **0** again, but add **1** on the left|
+|**1**0        | So we start back at **0** again, but carry **1** on the left|
```
Comment (Shivansh2407) [inline docs/binary.md:51]: Make this `but carry 1 on the left` bold as done in other tables to make it consistent

File: docs/binary.md (hunk 1 above)
Comment (abhinavsri360) [reply docs/binary.md]: made 0 and 1 bold as a where required

File: docs/binary.md (hunk 2 above)
Comment (abhinavsri360) [reply docs/binary.md]: Removed the redundant line

File: docs/binary.md (hunk 3 above)
Comment (abhinavsri360) [reply docs/binary.md:50]: done

**Comment by abhinavsri360** [comment]
made changes as requested. Please review @Shivansh2407

**Comment by abhinavsri360** [comment]
Thank you. I am looking forward to contribute to this book and other projects mentioned for GSoD '20. 🛩️

**Comment by YashKumarVerma** [comment]
I'd recommend not writing everything in bold in the body :confused:

**Comment by abhinavsri360** [comment]
That's the description in bold isn't it? 😅

**Comment by abhinavsri360** [comment]
This Pull Request Closes #45 
Please close the linked issue @YashKumarVerma

**Pull request merged**